postman/collection-dir/stripe/Payments/Session Token/event.test.js

```diff
@@ -9,7 +9,7 @@ pm.test("[POST]::/payments/session_tokens - Content-Type is application/json", f
 });
 
 const responseJson = pm.response.json();
-
+console.log(responseJson)
```

Comment:
Do we absolutely need this?

Reply:
Oh, it was added for debugging ApplePay related PR, will remove that